see-also.md

```diff
@@ -0,0 +1,17 @@
+# See Also
```

Comment:
It's good to separate out sections, but I think that the particular subset of references that are referenced directly in the Proposal.md should be included in there regardless of having this as a separate document.

This is the same concept of footnotes. And, since those are separate documents, it is possible for some to print out the Proposal.md file and ignore the rest, which would break the norms of proper attribution because of our short-sight. If they print "rendered" and the links are merely underlines, that is not on us, but not clearly making it obvious that a section of footnotes is in the original digital file is on us.

Reply:
These aren’t footnotes or attributions, I have those in each file. This is just the list of related documents/projects.